src/index.js

```diff
@@ -1,13 +1,5 @@
+import csv from './csv'
 import * as utils from './utils'
-import results from './results'
-import terms_stats from './terms_stats'
-import * as schemaToCSVTransforms from './schemaToCSVTransforms'
-import * as csv from './fast-csv-wrapper'
+import * as strategies from './strategies'
 
-export default {
-  ...utils,
-  ...schemaToCSVTransforms,
-  results,
-  terms_stats,
-  csv
-}
+export { strategies, csv, utils }
```

Comment:
We should flatten this - there are so few exports that keeping them nested like this is just inconvenient. "strategies" won't mean anything anymore, the whole lib is just csv, results, terms_stats, and utils.

Reply:
I found the flattened version so confusing as a user, especially when I first looked at it, can we keep them like this?